exchange/exchange.go

```diff
@@ -109,12 +120,33 @@ func (e *exchange) getAllBids(ctx context.Context, liveAdapters []openrtb_ext.Bi
 			// Structure to record extra tracking data generated during bidding
 			ae := new(seatResponseExtra)
 			ae.ResponseTimeMillis = int(elapsed / time.Millisecond)
+			// Timing statistics
+			e.m.AdapterMetrics[aName].RequestTimer.UpdateSince(start)
 			serr := make([]string, len(err))
 			for i := 0; i < len(err); i++ {
 				serr[i] = err[i].Error()
+				// NOTE, for a bidder that return multiple errors, we will log multiple errors for that request
+				// in the metrics. Need to remember that in analyzing the data.
```

Comment:
Could you also stick a `TODO #142:` note here? It'll help us find this place in the code when we go to fix that issue.

It's out of scope for this metrics project... but in the end, these metrics probably shouldn't include user input errors. For example, right now the Appnexus adapter returns an error when given Native/Audio `Imp`s because it doesn't support them. That doesn't mean there's a problem with the adapter, or anything an engineer should worry about... but today it will still get logged as an "error" in the metrics.

Reply:
WIll be in the next commit.